src/ui/component/tags/view.jsx

```diff
@@ -0,0 +1,52 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import { useTransition, animated } from 'react-spring';
+
+// We need to hard code a tag height so we can animate the tags reordering when a user changes them through
+// the drag and drop interface
+const TAG_HEIGHT = 20;
+
+type Props = {
+  tags: Array<Tag>,
+  className?: string,
```

Comment:
Mildly better if anything passing a class name could simply wrap the component or if there are distinct, enumerable styles to allow passing rather than this pattern (anything passing this is now complected with the component).

Reply:
Above comment still stands, especially since you're only using it in one place